server/controllers/ChallengeController.js

```diff
@@ -1,10 +1,37 @@
 const db = require("../models/index");
 const Challenge = db.Challenge;
 
-exports.get = async (req, res) => {
+exports.findOne = async (req, res) => {
 	try {
-        const challenges = await Challenge.findAll();
-        res.status(200).json(challenges);
+        const challenge_id = req.params.id;
+
+
+        // const challenge = await Challenge.findOne({
+        //     where: { 
+        //         id: challenge_id ,
+        //         // userId: req.user.id //Set up properly after done authentication
+        //     },
+        // });
+
+        //FOR TESTING
```

Comment:
I am assuming when calling this endpoint we currently return this test challenge object right?
Endpoint being `localhost:8080/api/challenges/1`?

Reply:
yes, it will return that object for any id at the moment, meaning `localhost:8080/api/challenges/1` and `localhost:8080/api/challenges/23` will return the same object as of right now.